container.go

```diff
@@ -824,9 +824,9 @@ func (c *Container) removeDrive() (err error) {
 	return nil
 }
 
-func (c *Container) attachDevices() error {
+func (c *Container) attachDevices(create bool) error {
```

Comment:
Isn't `container.state.State` enough to understand if the container is created or if it's running already ?